### PR TITLE
PMM-8266 remove prometheus files from tests

### PR DESCRIPTION
**Pull request by nikita-b**
[PMM-8266](https://jira.percona.com/browse/PMM-8266)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1856

- [ ] Links to other linked pull requests (optional).
---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

File: services/supervisord/logs.go [hunk 1]
```diff
@@ -153,16 +153,13 @@ func (l *Logs) files(ctx context.Context) []fileContent {
 		"/etc/nginx/conf.d/pmm-ssl.conf",
 
 		"/srv/alertmanager/alertmanager.base.yml",
-		"/srv/prometheus/prometheus.base.yml",
```
Comment (BupycHuk) [inline services/supervisord/logs.go:156]: wow wow wow, we still need this file to be existed.

Comment (nikita-b) [reply services/supervisord/logs.go:156]: What are we using it for?

Comment (BupycHuk) [reply services/supervisord/logs.go:156]: extending our config with user one, debugging

Comment (BupycHuk) [reply services/supervisord/logs.go:156]: https://www.percona.com/blog/2020/03/23/extending-pmm-prometheus-configuration/

Comment (nikita-b) [reply services/supervisord/logs.go:156]: But we don't have Prometheus in the container anymore...Or we use it also for something else?

Comment (BupycHuk) [reply services/supervisord/logs.go:156]: we have victoria metrics which is a replacement for prometheus.

Comment (atymchuk) [reply services/supervisord/logs.go:156]: yes, we need this one for sure ))

**Review by BupycHuk — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**